packages/database/src/migrations/20201208073839-WishFijiGovSurveysAssociateToCatchments-modifies-data.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+// [code, name]
+const oldCatchment = ['FJ_CATCH_Namosi', 'Namosi Catchment'];
+const newCatchment = ['FJ_CATCH_Upper_Navua', 'Upper Navua Catchment'];
+
+const originalEntityId = '5df8072d61f76a485c2eb9f3'; // CWMH Suva Hospital
+
+const updateCatchment = (db, code, updateCode, updateName) =>
+  db.runSql(`
+    update entity 
+    set code = '${updateCode}', name = '${updateName}' 
+    where code = '${code}';
+  `);
+
+const catchmentNameToCode = {
```

Comment:
I guess no more catchments are expected to be added till this runs - in which case, this hardcoded map is fine!

Alternatively, we could use the database as the ultimate source of truth